tests/features/persistent-urls.feature

```diff
@@ -39,8 +39,3 @@ Feature: Persistent URLs
     When I update alias of "News 1" node to "/alias1"
     And I go to "/news-2"
     Then I should see a persistent link for the node "News 1" pointing to "/en/alias1"
-    # Check link to first node with alias for another language.
-    When I update alias of "News 1" node to "/alias1_fr" for "French"
-    And I update alias of "News 2" node to "/news2_fr" for "French"
```

Comment:
News is untranslatable because "At least one field needs to be translatable to enable News for translation.". So this step fails.